presto-main/src/main/java/com/facebook/presto/Session.java

```diff
@@ -269,7 +270,9 @@ public Session beginTransactionId(TransactionId transactionId, TransactionManage
                 continue;
             }
             ConnectorId connectorId = transactionManager.getOptionalCatalogMetadata(transactionId, catalogName)
-                    .orElseThrow(() -> new PrestoException(NOT_FOUND, "Catalog does not exist: " + catalogName))
+                    .orElseThrow(() ->
```

Comment:
This formatting is strange. Let's write it like
```java
.orElseThrow(() -> new PrestoException(NOT_FOUND, format(
        "Catalog does not exist for session property: %s.%s",
        catalogName,
        catalogProperties.keySet().iterator().next())));

Reply:
Alternatively, we could simply change the error message but not include the name of the session property, which isn't relevant to the error:
```java
.orElseThrow(() -> new PrestoException(NOT_FOUND, "Catalog does not exist for session property: " + catalogName))

Reply:
Let's change the error message to this:
```java
"Session property catalog does not exist: "